linux/sound.cpp

```diff
@@ -57,42 +61,61 @@ void CSound::OpenJack ( const bool  bNoAutoJackConnect,
     // check sample rate, if not correct, just fire error
     if ( jack_get_sample_rate ( pJackClient ) != SYSTEM_SAMPLE_RATE_HZ )
     {
-        throw CGenErr ( tr ( "The Jack server sample rate is different from "
-            "the required one. The required sample rate is:" ) + " <b>" +
-            QString().setNum ( SYSTEM_SAMPLE_RATE_HZ ) + " Hz</b>. " + tr ( "You can "
-            "use a tool like <i><a href=""http://qjackctl.sourceforge.net"">QJackCtl</a></i> "
-            "to adjust the Jack server sample rate." ) + "<br>" + tr ( "Make sure to set the "
-            "Frames/Period to a low value like " ) +
+        throw CGenErr (
+            tr ( "The Jack server sample rate is different from "
+                 "the required one. The required sample rate is:" ) +
+            " <b>" + QString().setNum ( SYSTEM_SAMPLE_RATE_HZ ) + " Hz</b>. " +
+            tr ( "You can "
+                 "use a tool like <i><a href="
+                 "http://qjackctl.sourceforge.net"
+                 ">QJackCtl</a></i> "
+                 "to adjust the Jack server sample rate." ) +
+            "<br>" +
+            tr ( "Make sure to set the "
+                 "Frames/Period to a low value like " ) +
             QString().setNum ( DOUBLE_SYSTEM_FRAME_SIZE_SAMPLES ) +
             tr ( " to achieve a low delay." ) );
     }
 
     // create four ports (two for input, two for output -> stereo)
-    input_port_left = jack_port_register ( pJackClient, "input left",
-        JACK_DEFAULT_AUDIO_TYPE, JackPortIsInput, 0 );
-
-    input_port_right = jack_port_register ( pJackClient, "input right",
-        JACK_DEFAULT_AUDIO_TYPE, JackPortIsInput, 0 );
-
-    output_port_left = jack_port_register ( pJackClient, "output left",
-        JACK_DEFAULT_AUDIO_TYPE, JackPortIsOutput, 0 );
-
-    output_port_right = jack_port_register ( pJackClient, "output right",
-        JACK_DEFAULT_AUDIO_TYPE, JackPortIsOutput, 0 );
-
-    if ( ( input_port_left   == nullptr ) ||
-         ( input_port_right  == nullptr ) ||
-         ( output_port_left  == nullptr ) ||
-         ( output_port_right == nullptr ) )
+    input_port_left = jack_port_register ( pJackClient,
```

Comment:
The original was accepted layout.  Is there any way to teach that to clang?

Reply:
I don't think there's a way to align by `||`